rotkehlchen/chain/ethereum/modules/convex/decoder.py

```diff
@@ -176,7 +177,7 @@ def _decode_gauge_events(self, context: DecoderContext) -> DecodingOutput:
             ):
                 if context.tx_log.topics[0] in WITHDRAWAL_TOPICS:
                     event.event_type = HistoryEventType.WITHDRAWAL
-                    event.event_subtype = HistoryEventSubType.REMOVE_ASSET
+                    event.event_subtype = HistoryEventSubType.REDEEM_WRAPPED if found_return_wrapped else HistoryEventSubType.REMOVE_ASSET  # noqa: E501
```

Comment:
this one I've checked and indeed you can lock cvxCRV and you don't receive anything in return